types/lodash-es/index.d.ts

```diff
@@ -4,304 +4,305 @@
-export { default as subtract } from "./subtract";
-export { default as sum } from "./sum";
-export { default as sumBy } from "./sumBy";
-export { default as tail } from "./tail";
-export { default as take } from "./take";
-export { default as takeRight } from "./takeRight";
-export { default as takeRightWhile } from "./takeRightWhile";
-export { default as takeWhile } from "./takeWhile";
-export { default as template } from "./template";
-export { default as templateSettings } from "./templateSettings";
-export { default as throttle, ThrottleSettings } from "./throttle";
-export { default as times } from "./times";
-export { default as toArray } from "./toArray";
-export { default as toFinite } from "./toFinite";
-export { default as toInteger } from "./toInteger";
-export { default as toLength } from "./toLength";
-export { default as toLower } from "./toLower";
-export { default as toNumber } from "./toNumber";
-export { default as toPairs } from "./toPairs";
-export { default as toPairsIn } from "./toPairsIn";
-export { default as toPath } from "./toPath";
-export { default as toPlainObject } from "./toPlainObject";
-export { default as toSafeInteger } from "./toSafeInteger";
-export { default as toString } from "./toString";
-export { default as toUpper } from "./toUpper";
-export { default as transform } from "./transform";
-export { default as trim } from "./trim";
-export { default as trimEnd } from "./trimEnd";
-export { default as trimStart } from "./trimStart";
-export { default as truncate } from "./truncate";
-export { default as unary } from "./unary";
-export { default as unescape } from "./unescape";
-export { default as union } from "./union";
-export { default as unionBy } from "./unionBy";
-export { default as unionWith } from "./unionWith";
-export { default as uniq } from "./uniq";
-export { default as uniqBy } from "./uniqBy";
-export { default as uniqWith } from "./uniqWith";
-export { default as uniqueId } from "./uniqueId";
-export { default as unset } from "./unset";
-export { default as unzip } from "./unzip";
-export { default as unzipWith } from "./unzipWith";
-export { default as update } from "./update";
-export { default as updateWith } from "./updateWith";
-export { default as upperCase } from "./upperCase";
-export { default as upperFirst } from "./upperFirst";
-export { default as values } from "./values";
-export { default as valuesIn } from "./valuesIn";
-export { default as without } from "./without";
-export { default as words } from "./words";
-export { default as wrap } from "./wrap";
-export { default as xor } from "./xor";
-export { default as xorBy } from "./xorBy";
-export { default as xorWith } from "./xorWith";
-export { default as zip } from "./zip";
-export { default as zipObject } from "./zipObject";
-export { default as zipObjectDeep } from "./zipObjectDeep";
-export { default as zipWith } from "./zipWith";
+export { default as add } from './add';
```

Comment:
was there a reason to change file formatting/style?

if not please revert and leave only actual changes

Reply:
Please revert the quotation mark changes unless they came from prettier.